packages/bytes/src/bytes.cairo

```diff
@@ -116,8 +117,18 @@ pub trait BytesTrait {
     /// concat with other Bytes
     fn concat(ref self: Bytes, other: @Bytes);
     /// keccak hash
+    #[deprecated(
+        feature: "deprecated-keccak",
+        note: "Use `core::keccak::compute_keccak_byte_array`.",
+        since: "2.7.0"
+    )]
     fn keccak(self: @Bytes) -> u256;
     /// sha256 hash
+    #[deprecated(
```

Comment:
shouldnt be deprecated. `Bytes` != `ByteArray` - and if we consider this deprecated then the whole Bytes type should be deprecated.

Reply:
`sha256(self: @Bytes)` should use the `sha256` syscall

Reply:
Anyway since it is using a deprecated fn (math::sha256) the deprecate will propagate upwards.
Maybe we can make this whole bytes stuff deprecated indeed

Reply:
Should we make the whole Byte deprecated?